Xamarin.Forms.Core/DeviceInfo.cs

```diff
@@ -22,6 +23,18 @@ public DeviceOrientation CurrentOrientation
 			}
 		}
 
+		public FlowDirection CurrentFlowDirection
+		{
+			get { return _currentFlowDirection; }
+			set
+			{
+				if (Equals(_currentFlowDirection, value))
+					return;
+				_currentFlowDirection = value;
+				OnPropertyChanged();
```

Comment:
Is this ever expected to change? I might be wrong, but the samples aren't listening to INPC for this. So the proper use would be to bind to this value ?

Reply:
I expected app users to be able to change the language/locale without restarting the app, but that doesn't appear to actually be possible. Android is the closest, but it does restart the activity. I'll remove this.